modules/sno-adding-worker-nodes-to-sno-clusters-manually.adoc

```diff
@@ -79,22 +79,58 @@ $ curl -L $ISO_URL -o rhcos-live.iso
 
 . Use the {op-system} ISO and the hosted `worker.ign` file to install the worker node:
 
-.. Boot the target host using the {op-system} ISO and your preferred method of installation.
+.. Boot the target host with the {op-system} ISO and your preferred method of installation.
 
-.. When the target host has booted from the {op-system} ISO, open a console on the target host, and run the following `coreos-installer` command to install the system:
+.. When the target host has booted from the {op-system} ISO, open a console on the target host.
+
+.. If your local network does not have DHCP enabled, you need to specify the static IP of the worker node before running the {op-system} installation. For example, run the following commands on the target host console:
+
+... Configure the worker host network connection with a static IP:
 +
 [source,terminal]
 ----
-$ coreos-installer install --ignition-url=<hosted_worker_ign_file> <hard_disk>
+$ nmcli con mod <network_interface> ipv4.method manual /
+ipv4.addresses <static_ip>/<cidr> ipv4.gateway <network_gateway> ipv4.dns <dns_server>
+----
++
+where:
++
+--
+<static_ip>:: Is the host static IP address, for example, `10.1.101.50`
```

Comment:
we don't mention the cidr. Maybe:
<static_ip>/<cidr>:: Is the host static IP address and CIDR, for example, `10.1.101.50/24`